src/main/java/roomescape/config/PaymentClientResponseErrorHandler.java

```diff
@@ -0,0 +1,37 @@
+package roomescape.config;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.stream.Collectors;
+
+import org.springframework.http.client.ClientHttpResponse;
+import org.springframework.stereotype.Component;
+import org.springframework.web.client.ResponseErrorHandler;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import roomescape.exception.PaymentException;
+
+@Component
+public class PaymentClientResponseErrorHandler implements ResponseErrorHandler {
+
+    private ObjectMapper objectMapper = new ObjectMapper();
+
+
```

Comment:
```suggestion
    private final ObjectMapper objectMapper = new ObjectMapper();

```
bean으로 등록된 `ObjectMapper`를 사용하는 것이 아닌 생성하는 이유가 있을까요?

Reply:
특별한 이유가 있는 것이 아닌 단순 실수로 해당 부분만 `ObjectMapper`를 생성한 것 같습니다
주의하겠습니다!